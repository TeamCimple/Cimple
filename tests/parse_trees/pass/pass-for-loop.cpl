int main() {
	int i;
	int j;
	
	for(i = 0; i < 100; i++) {
		printf("AYEAYEAYE");
	}
	return 0;
}
