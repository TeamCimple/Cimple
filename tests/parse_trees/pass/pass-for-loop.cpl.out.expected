FuncDecl(Name: Var(main) ReturnType: int Parameters:  Body: CompoundStatement(Declarations: (int Var(i),), (int Var(j),),  StatementList: FOR EQUALS(Identifier(i), Int(0)) Compare(Identifier(i),LESS_THAN,Int(100)) Postfix(Identifier(i),++) CompoundStatement(Declarations:  StatementList: Statement(Call(Receiver()FunctionName: Identifier(printf) Params: String("AYEAYEAYE")))), RETURN(Int(0)))) 
